src/Tizen.NUI/src/public/BaseComponents/Style/Selector.cs

```diff
@@ -27,10 +28,31 @@ namespace Tizen.NUI.BaseComponents
     /// <since_tizen> 6 </since_tizen>
     /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class Selector<T> : StateValueCollection<T>
+    public class Selector<T>
     {
         private readonly bool cloneable = typeof(T).IsAssignableFrom(typeof(ICloneable));
 
+        /// <summary>
+        /// The list for adding state-value pair.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IList<StateValuePair<T>> StateValueList { get; private set; } = new List<StateValuePair<T>>();
```

Comment:
Why don't we just use List instead of IList?

Reply:
MS design-guidelines not recommend `List<T>`.
```
❌ DO NOT use ArrayList or List<T> in public APIs.

These types are data structures designed to be used in internal implementation, not in public APIs. 
List<T> is optimized for performance and power at the cost of cleanness of the APIs and flexibility. 
For example, if you return List<T>, you will not ever be able to receive notifications when client code modifies the collection. 
Also, List<T> exposes many members, such as BinarySearch, that are not useful or applicable in many scenarios. 
The following two sections describe types (abstractions) designed specifically for use in public APIs.
```
(https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections)